packages/js/tsup.config.ts

```diff
@@ -39,24 +39,24 @@ export default defineConfig([
     format: 'cjs',
     outDir: 'dist/cjs',
   },
-  {
-    ...baseConfig,
-    entry: { novu: 'src/umd.ts' },
-    format: ['iife'],
-    minify: true,
-    dts: false,
-    outExtension: () => {
-      return {
-        js: '.min.js',
-      };
-    },
-    esbuildPlugins: [
-      compress({
-        gzip: true,
-        brotli: false,
-        outputDir: '.',
-        exclude: ['**/*.map'],
-      }),
-    ],
-  },
+  // {
+  //   ...baseConfig,
+  //   entry: { novu: 'src/umd.ts' },
+  //   format: ['iife'],
+  //   minify: true,
+  //   dts: false,
+  //   outExtension: () => {
+  //     return {
+  //       js: '.min.js',
+  //     };
+  //   },
+  //   esbuildPlugins: [
+  //     compress({
+  //       gzip: true,
+  //       brotli: false,
+  //       outputDir: '.',
+  //       exclude: ['**/*.map'],
+  //     }),
+  //   ],
+  // },
```

Comment:
Commented out this code temporarily until we investigate why Rollup includes in UMD build some additional code that results in bigger bundle size.